README.md

```diff
@@ -2,17 +2,7 @@ easy-solr4files-index
 ============================
 [![Build Status](https://travis-ci.org/DANS-KNAW/easy-solr4files-index.png?branch=master)](https://travis-ci.org/DANS-KNAW/easy-solr4files-index)
 
-* [SYNOPSIS](#synopsis)
```

Comment:
@rkruithof : actually, by removing the TOC here, we are left without one, as this project doesn't have a gh-pages site yet. I'll merge it anyway and create an issue for converting al OSS-modules to using gh-pages.